problemsets-mat223/modules/module5.tex

```diff
@@ -155,15 +192,31 @@
 \begin{example}
 	Compute the projection of $\vec a=\mat{3\\7}$ onto $\ell=\Span\Set*{\mat{1\\-4}}$.
 
-	XXX Finish
+	By the theorem above, let $\vec b = \mat{1\\-4}$, we have
```

Comment:
You cannot "let" by the theorem above. You must "let" first. Then in a new sentence, you can explain the consequences of the theorem